src/main/kotlin/gropius/model/template/MutableTemplatedNode.kt

```diff
@@ -0,0 +1,12 @@
+package gropius.model.template
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.annotations.GraphQLIgnore
+
+@GraphQLDescription("Interface for all types which support templates that can be written by the user.")
+interface MutableTemplatedNode : TemplatedNode {
+    @GraphQLIgnore
+    private fun graphQLKotlinDummyFunction() {
+        // Dummy function to avoid GraphQL kotlin optimization weirdness
```

Comment:
maybe change this to a KDoc comment

Reply:
627fa894d0487f997628d7d8f967a37420cce136?